include/mbedtls/aes.h

```diff
@@ -60,7 +60,11 @@
 
 /* Error codes in range 0x0021-0x0025 */
 #define MBEDTLS_ERR_AES_BAD_INPUT_DATA                    -0x0021  /**< Invalid input data. */
+
+/* MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE is deprecated and should not be used. */
```

Comment:
Not a blocker because there's precedent, but why have the statement on the symbol being deprecated above it and _not in the doxygen comment_?

Reply:
Not sure what the best way for our library to document or tell the compiler that a define is deprecated, yet. Copied the approach used previously.

Reply:
If there is a deprecated feature for doxygen, I think it is better. The approach use previously was on a file that is not part of the doxygen headers.